### ISSUE TITLE
Pager is off, causing multi page pdfs to be cut off at the top

### ISSUE THREAD

**Issue by justus-lattice** [pdfme/pdfme#155]
### Describe the bug

If you have a multi-page pdf, and you page through the pages, you will notice for each one you page through, the offset keeps on growing in size, and therefore the pages don't start where they should.

### To Reproduce

Upload a 10 page doc (I just generated 10 blank pages from a google doc), upload it here: https://pdfme.com/template-design and use the pager and notice the offset keeps growing and each subsequent page gets slowly cut off at the top.

### Expected behavior

Each page should start at the same place when using the pager.

### Your Environment

```markdown
Using https://pdfme.com/template-design

- Operating system: Mac 12.6.3
- Node.js version or Browser name & version: Chrome 111.0.5563.146
```

### Your Error Log

```shell
N/A
```

**Comment by hand-dot** [comment]
Hi @justus-lattice !

Thank you for reporting the bug.

If possible, could you please fix it?

The problem you identified is correct and I would be happy to review and merge it as soon as you submit a fixed PR, but I don't have time to fix it.

**Comment by peteward** [comment]
There are a couple of issues here, we crop previous pages, and the mask is not sitting in the correct position.
Here is an example:

![Screenshot 2023-10-26 at 15 13 35](https://github.com/pdfme/pdfme/assets/7068515/1e931125-6c24-4c71-83ae-7381d6fb7eff)

Compared to the PDF:

![Screenshot 2023-10-26 at 15 13 50](https://github.com/pdfme/pdfme/assets/7068515/4050e6e4-26f2-4bb8-803a-b6a880ca73bd)

I will try to fix this now.

**Comment by peteward** [comment]
This has been missed previously because the common example we use has a white background with no content near the bottom - it was easy to miss this cropping.

**Comment by peteward** [comment]
I have fixed this locally, and will open a PR when #289 is merged.

![Screenshot 2023-10-26 at 16 41 57](https://github.com/pdfme/pdfme/assets/7068515/cd272a54-d520-4586-84a7-23437e9c5112)
